sql/1.096/dynamic-blocks.sql

```diff
@@ -0,0 +1,78 @@
+BEGIN;
+drop table if exists instant_answer_blocks;
+alter table instant_answer drop column if exists blockgroup;
+drop table if exists instant_answer_blockgroup;
+
+create table instant_answer_blockgroup (
+    id serial primary key,
+    blockgroup varchar(20) not null unique
+);
+
+copy instant_answer_blockgroup (blockgroup) from stdin;
+goodie_exlusive
+spice_killresult
+spice_nonexclusive
+goodie
+spice
+fathead
+longtail
+\.
+
+
+alter table instant_answer
+    add column blockgroup varchar(20) references instant_answer_blockgroup (blockgroup);
+    
+update instant_answer set blockgroup = 'goodie' where repo = 'goodies';
```

Comment:
@zachthompson I think probably here I get this error:
ERROR:  insert or update on table "instant_answer" violates foreign key constraint "instant_answer_blockgroup_fkey"
DETAIL:  Key (blockgroup)=(0) is not present in table "instant_answer_blockgroup".